.phpcs.xml.dist

```diff
@@ -89,6 +89,9 @@
 			<!-- Don't trigger on the main file as renaming it would deactivate the plugin. -->
 			<property name="excluded_files_strict_check" type="array">
 				<element value="duplicate-post.php"/>
+				<!-- Don't trigger on test bootstrap files. -->
+				<element value="tests/integration/bootstrap.php"/>
+				<element value="tests/unit/bootstrap.php"/>
```

Comment:
Are these really necessary ? What error are you getting ?
It may be better to put these as error code specific excludes in the "Selective excludes" section.

Reply:
I get
```
Files containing function declarations should have "-functions" as a suffix. Expected "bootstrap-functions.php", but found "bootstrap.php". (Yoast.Files.FileName.InvalidFunctionsFileName)
```

But that's only for the integrations bootstrap, so I'll switch to a selective exclude